### ISSUE TITLE
Generated command for wget/curl incorrectly duplicates cookies

### ISSUE THREAD

**Issue by m-dierker** [a85/POSTMan-Chrome-Extension#8170]
**Describe the bug**
Exporting a request to replicate in `curl` or `wget` incorrectly duplicates cookies.

**To Reproduce**
1. Put in any URL for a GET request (ex: `https://google.com`)
2. Add a Header with the key `Cookie` and the value `blah`
3. Click Send more than once.
4. Attempt to get cURL code via `Code --> cURL`.
5. Observe that `blah` is included twice.

**Expected behavior**
The output of `Code --> cURL` should give a command that exactly replicates the Postman request. The cookie `blah` should only be included once.

**Actual Output (observe incorrect blah,blah)**
```
curl -X GET \
  https://google.com \
  -H 'Accept: */*' \
  -H 'Accept-Encoding: gzip, deflate' \
  -H 'Cache-Control: no-cache' \
  -H 'Connection: keep-alive' \
  -H 'Cookie: blah,blah; 1P_JAR=2020-03-11-22; NID=200=XRYon05k9-_FKVxUo52aHKEixBuXICYrdQ8wgw_rAQt4Me-s6VJMLh7JHPEPxjnfYaKw82iUwzF43lvVYpdGJUpmZgiG7LTfkq_7X4InAaEEeLdAFOB6H9xAlqbRBHEaLB7lMjl0BgQ8UKhjCHcYjGJWZr0FQZjcx3GEGZd2dmA' \
  -H 'Postman-Token: 5b89950f-5e94-4d65-b733-2482233510b8,6aa23da2-b4d6-467f-b576-53ce135daf3d' \
  -H 'Referer: https://google.com/' \
  -H 'User-Agent: PostmanRuntime/7.18.0' \
  -H 'cache-control: no-cache'
```

**App information (please complete the following information):**
 - App Type: Native
 - Postman Version: 7.9.0
 - OS: macOS Catalina 10.15.3

**Additional context**
It took me a bunch of time to figure out why the cURL command was giving different results than Postman. This would be helpful to fix. :D

**Comment by shreys7** [comment]
I followed the steps to reproduce and can see the cookie value being added only once in the generated cURL snippet. Can you check whether the setting `New Code Generation Mode` is enabled in global settings and you are on the latest version of the App?

![image](https://user-images.githubusercontent.com/20665864/85123211-7d20a380-b245-11ea-9ba8-c4d53071bf5d.png)

**Comment by m-dierker** [comment]
Looks like this was fixed sometime between 7.9.0 and 7.26.1. Thanks!
